gramps_webapi/api/resources/base.py

```diff
@@ -63,6 +70,7 @@ class GrampsObjectResource(GrampsObjectResourceHelper, Resource):
             ),
             "profile": fields.Str(validate=validate.Length(equal=0)),
             "extend": fields.DelimitedList(fields.Str(validate=validate.Length(min=1))),
+            "backlinks": fields.Boolean(missing=False),
```

Comment:
It's true that it doesn't make sense to be inconsistent, so we should use the same everywhere.

The reason I used the built-in boolean field type was that I was feeling uneasy about treating an empty string as "true". I even found that there is a [discussion](https://github.com/marshmallow-code/marshmallow/issues/713) to treat an empty string as "none". I mean, I don't claim that the way we do it now will break since we just check if the value is present in the query string. But it somehow doesn't feel entirely clean.

For instance, by forcing the "user" (or rather, apps) to use `something=1` and `something=0`, (or `true`/`false`), it would also be possible to change the default from false to true at some point. It somehow seems cleaner.

Reply:
That is a good point about the default that I had not thought of. That makes good sense to me, I agree. I will submit something to switch the ones I had changed back to booleans.